src/webextension/firstrun/components/pages.js

```diff
@@ -88,6 +92,8 @@ export class VerifyPage2 extends React.Component {
         error: "wrong password",
       });
     }
+    telemetry.recordEvent("lockbox", "render", "confirmPWPage");
+
```

Comment:
This is getting fired when the user *clicks* to confirm their FxA password; however, the spec says that it "fires when the screen asking for password confirm *renders*." (emphasis mine) If you want to conform to the spec, you'd probably want to add a method called `componentDidMount()` and run this code there.

Also, nit: remove the extra newline

Reply:
click will still tell us that they got this far in the flow, which is all this event is trying to accomplish. I'll change the spec accordingly